dotnet/Library/Okapi/Okapi.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <AssemblyName>Okapi.Net</AssemblyName>
     <PackageId>Okapi.Net</PackageId>
-    <TargetFrameworks>netstandard2.1;xamarinios10</TargetFrameworks>
```

Comment:
This won't work I'm afraid. The reason why we need separate compile target for xamarin is because of an `#if __IOS__` directive that is specific to iOS targets. `net50` won't support iOS targets until 6.0.

Reply:
Yes, otherwise it won't get loaded by name.